workspace/src/localization/localization_py/localization_py/EKF.py

```diff
@@ -0,0 +1,174 @@
+import numpy as np
```

Comment:
Open to discussion, but I think proper docstring would be nice. So at the bare minimum, docstring describing class purpose and what a method is doing. Again, open for discussion.

Reply:
How do the current docstrings look? Should they be more/less descriptive than they currently are?

Reply:
More descriptive. Like this: https://github.com/uwsbel/autonomy-toolkit/blob/main/autonomy_toolkit/utils/atk_config.py. I prefer google docstrings. Basically, overview of what the method is doing, maybe some background of the math/idea, if needed, and documenting the purpose of inputs/outputs (if any).